group_vars/mash_servers

```diff
@@ -676,6 +690,66 @@ focalboard_container_labels_traefik_tls_certResolver: "{{ devture_traefik_certRe
 
 
 
+########################################################################
+#                                                                      #
+# funkwhale                                                            #
+#                                                                      #
+########################################################################
+
+funkwhale_enabled: false
+
+funkwhale_identifier: "{{ mash_playbook_service_identifier_prefix }}funkwhale"
+
+funkwhale_base_path: "{{ mash_playbook_base_path }}/{{ mash_playbook_service_base_directory_name_prefix }}funkwhale"
+
+funkwhale_uid: "{{ mash_playbook_uid }}"
+funkwhale_gid: "{{ mash_playbook_gid }}"
+
+funkwhale_database_hostname: "{{ devture_postgres_identifier if devture_postgres_enabled else '' }}"
+funkwhale_database_port: "{{ '5432' if devture_postgres_enabled else '' }}"
+funkwhale_database_password: "{{ '%s' | format(mash_playbook_generic_secret_key) | password_hash('sha512', 'db.funkwhale', rounds=655555) | to_uuid }}"
+funkwhale_database_username: "{{ funkwhale_identifier }}"
+
+funkwhale_systemd_required_services_list: |
+  {{
+    (['docker.service'])
+    +
+    ([devture_postgres_identifier ~ '.service'] if devture_postgres_enabled and funkwhale_database_hostname == devture_postgres_identifier else [])
+  }}
```

Comment:
I don't see `funkwhale_systemd_required_services_list` in the role.

There are these though:
- `funkwhale_api_systemd_required_services_list`
- `funkwhale_frontend_systemd_required_services_list`

I think it'd be better to use `_default`, `_auto` and `_custom` variables for each of these and to override the `_auto` variables here (`funkwhale_api_systemd_required_services_list_auto` and `funkwhale_frontend_systemd_required_services_list_auto` to inject Postgres and to not bother with `docker.service` which will be part of the `_default` set).

Reply:
Fixed :+1: